docs/tutorials/getting_started/getting_started.rst

```diff
@@ -0,0 +1,53 @@
+Getting started
+===============
+
+lipsum
+
+
+Installation
+------------
+
+The simplest way to install the package is to obtain it from the PyPi
+repository::
+
+    $ pip install axelrod
+
+
+You can also build it from source if you would like to::
+
+    $ git clone https://github.com/Axelrod-Python/Axelrod.git
+    $ cd Axelrod
+    $ python setup.py install
+
+If you do this you will need to also install the dependencies::
+
+    $ pip install -r requirements.txt
+
+Creating and running a simple tournament
+----------------------------------------
+
+The following lines of code create a simple list of strategies::
+
+    >>> import axelrod as axl
+    >>> strategies = [axl.Cooperator(), axl.Defector(),
+    ...               axl.TitForTat(), axl.Grudger()]
+    >>> strategies
+    [Cooperator, Defector, Tit For Tat, Grudger]
+
+We can now create a tournament, play it, saving the results and viewing the
+ranks of each player::
+
```

Comment:
<pedantic grammatical correction>This should be in the same tense and a single player has a single rank, not several - We can now create a tournament, play it, save the results and view the rank of each player::</pedantic grammatical correction>